stream-chat-android-core/src/main/java/io/getstream/chat/android/client/call/Call.kt

```diff
@@ -82,3 +83,20 @@ public fun <T : Any, K : Any> Call<T>.zipWith(call: Call<K>): Call<Pair<T, K>> {
 
 @InternalStreamChatApi
 public fun Call<*>.toUnitCall(): Call<Unit> = map {}
+
+private val onSuccessStub: (Any) -> Unit = {}
+private val onErrorStub: (ChatError) -> Unit = {}
+
+@InternalStreamChatApi
+public fun <T : Any> Call<T>.enqueue(
```

Comment:
🤔 Why it was added to `.api` file?

Reply:
Maybe because the annotation is not processed when the `.api` file is generated?
In the end, it is a public method that is accessible by the client

Reply:
This is a bug in the binary validator plugin - it doesn't realize that the function that it generated a secondary synthetic `$default` function for (to support its default params in the bytecode) was originally annotated with our `@InternalStreamChatApi` annotation.

I've opened an issue for it here: https://github.com/Kotlin/binary-compatibility-validator/issues/58